test-framework/core/pom.xml

```diff
@@ -48,6 +48,10 @@
                     <groupId>io.quarkus</groupId>
                     <artifactId>quarkus-junit5-config</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
Why is this being added?

Reply:
Testcontainers config from Infinispan is rewriting our testcontainers default configuration, when running db containers locally.